src/dndbeyond/content-scripts/character.js

```diff
@@ -319,6 +319,13 @@ function rollItem(force_display = false) {
             damage_types.push("Psychic");
             character.mergeCharacterSettings({ "bard-psychic-blades": false });
         }
+        //Protector Aasimar: Radiant Soul Damage
+        if (character.hasRacialTrait("Radiant Soul") &&
+            character.getSetting("protector-aasimar-radiant-soul", true)) {
+            damages.push(" + " + character._level);
```

Comment:
nitpick but no need for the "+" here, it's an independent damage of '3', there's nothing before it, so it doesn't need to be an equation.

Reply:
Fixed. I had based it of Great Weapon Master and Sharpshooter, so should those be changed as well?

![image](https://user-images.githubusercontent.com/7988297/81576466-858fdf80-9365-11ea-90a3-60717146429d.png)